src/uct/sm/mm/base/mm_ep.c

```diff
@@ -320,7 +321,8 @@ static UCS_F_ALWAYS_INLINE ssize_t uct_mm_ep_am_common_send(
     switch (send_op) {
     case UCT_MM_SEND_AM_SHORT:
         /* write to the remote FIFO */
-        uct_am_short_fill_data(elem + 1, header, payload, length);
+        uct_am_short_fill_data(elem + 1, header, payload, length,
```

Comment:
do we really have a case where length > 3/4 of Milan L3 cache (96MB CCD => 96*3/4)? else we might keep `uct_am_short_fill_data()` untouched.

Reply:
Yes, some workloads benefit with this as explained in the above comments.  User can use "-x UCX_NT_BUFFER_TRANSFER_MIN=0" in the commandline and use nt-buffer-transfer for the entire range of transfers.